lib/trino-parquet/src/main/java/io/trino/parquet/ParquetTypeUtils.java

```diff
@@ -251,6 +251,11 @@ public static boolean isValueNull(boolean required, int definitionLevel, int max
         return !required && (definitionLevel == maxDefinitionLevel - 1);
     }
 
+    public static boolean isOptionalValueNull(int definitionLevel, int maxDefinitionLevel)
```

Comment:
that is some cryptic name. Consider some more explanatory name + maybe comment

Reply:
Renamed to `isOptionalFieldValueNull`
Same code comment as the above method applies